docs/quickstart.md

```diff
@@ -67,65 +68,68 @@ kubectl get clusterpackageinstall -A
 and wait for them to be `Ready:True`
 ```
 NAMESPACE           NAME               READY   SOURCE                PACKAGE
-crossplane-system   provider-gcp       True    registry.upbound.io   crossplane/provider-gcp:v0.9.0
-crossplane-system   stack-gcp-sample   True    registry.upbound.io   crossplane/stack-gcp-sample:v0.5.0
+crossplane-system   provider-gcp       True    registry.upbound.io   crossplane/provider-gcp:v0.10.0
+crossplane-system   stack-gcp-sample   True    registry.upbound.io   crossplane/stack-gcp-sample:v0.6.0
 ```
 
-#### Create gcp-secret.json
-Run the following replacing `MY_GCP_PROJECT_ID` with your own:
+#### Get GCP Account Keyfile
 ```
-curl -s https://raw.githubusercontent.com/crossplane/stack-minimal-gcp/master/hack/gcp-credentials.sh | PROJECT_ID=[MY_GCP_PROJECT_ID] bash -
```

Comment:
can we not keep using the curl/bash approach which is makes all this 1 line essentially? i thought that was a much more clean and easy experience

Reply:
@jbw976 we moved away from this in the updated the crossplane.io docs as @negz was very against using the `curl | bash` since it's unlikely someone would run an arbitrary script to grant access to their account, and it also prints the secret into the console for all to see, and requires cut/paste that secret into a file, which isn't great.

Reply:
Echoing @prasek here. I would never run an arbitrary shell script that manipulated IAM roles and permissions in my project. Let's show folks what we're doing and help them learn, even if it is more CLI commands to paste.